### PR TITLE
Feedback for My Forest V2

### PR DESCRIPTION
**Pull request by mohitb35**
Addresses the following feedback for My Forest V2

1. Removes forced capitalization for profile name
2. Updates map credit tooltip styles, and repositions zoom controls/map credits in one column.
3. Removes border radius around profile background image, and uses a sharper background image
4. Standardizes border radius for all containers 
5. Hides the target on/off toggle switch if no target is entered
6. Resolves bug with clicking on Upload new profile image button
7. Displays a higher quality profile image version
8. Refactor to remove duplicate CustomTooltip components
9. Adds `showLeaderboard`/`exposeCommunity` functionality in the app
10. Wraps webapp button text properly
11. Minor tweaks to grid layout for responsiveness

### CODE REVIEW

**Review by mariahosfeld — COMMENTED** [review]
Changes look good to me so far.
Only issue: cannot check leaderboard functionality, due to already noted issue that needs to be resolved.

**Comment by sunilsabatp** [comment]
While accessing a public profile, I encountered this error in the console

image

**Review by sunilsabatp — CHANGES_REQUESTED** [review]

File: src/features/user/MFV2/CommunityContributions/index.tsx [hunk 1]
```diff
@@ -89,9 +89,11 @@ const CommunityContributions = ({
     <div className={styles.communityContributions}>
       <div className={styles.header}>
         <div className={styles.infoIcon}>
-          <CustomTooltip height={15} width={15} color={'#828282'}>
+          <CustomTooltip height={15} width={14} color={'#828282'}>
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/user/MFV2/CommunityContributions/index.tsx:92]: Done

File: src/features/user/MFV2/ForestProgress/TargetFormInput.tsx [hunk 2]
```diff
@@ -58,11 +58,13 @@ const TargetFormInput = ({
     >
       <div className={styles.switchContainer}>
         <TargetFormInputLabel dataType={dataType} />
-        <TargetSwitch
-          switchColor={targetColor(dataType) ?? ''}
-          checked={checked}
-          onChange={handleTargetSwitch}
-        />
+        {localTarget > 0 && (
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/user/MFV2/ForestProgress/TargetFormInput.tsx:61]: Done

**Comment by mohitb35** [comment]
> While accessing a public profile, I encountered this error in the console
> 
> image

I'm guessing this was on localhost? Did you try `npx prisma generate` ?

**Comment by sunilsabatp** [comment]
> > While accessing a public profile, I encountered this error in the console
> > image
> 
> I'm guessing this was on localhost? Did you try `npx prisma generate` ?

it's working now

**Review by sunilsabatp — APPROVED** [review]

**Pull request merged**